components/layout/Nav.js

```diff
@@ -4,22 +4,25 @@ import useUser from '../auth/User';
 import LogOut from '../auth/LogOut';
 import { useCart } from '../../context/cartState';
 import CartTally from '../cart/CartTally';
+import AccountDropdown from '../elements/AccountDropdown';
 
 const Nav = () => {
   const cart = useCart();
   const user = useUser();
 
   return (
     <StyledNav>
-      <Link href="/products">All Tutorials</Link>
       {user ? (
-        <>
-          <Link href="/my-tutorials">my tutorials</Link>
-          <Link href="/account">account</Link>
-          <LogOut />
+        <section>
+          <AccountDropdown>
+            <Link href="/my-tutorials">my tutorials</Link>
+            <Link href="/products">tutorials</Link>
+            <Link href="/account">account</Link>
+            <LogOut />
+          </AccountDropdown>
           {user.cart?.length > 0 && (
-            <button type="button" onClick={() => cart.openCart()}>
-              view cart
+            <button id="cart" type="button" onClick={() => cart.openCart()}>
```

Comment:
No need to set the `type` as `type="button"` is already the default. Set it when e.g. you need `type="submit"`